packages/material-ui/src/Tooltip/Tooltip.test.js

```diff
@@ -466,4 +466,22 @@ describe('<Tooltip />', () => {
       );
     });
   });
+
+  it('should use the same popper.js instance between two renders', () => {
+    const popperRef = React.createRef();
+    const { setProps } = render(
+      <Tooltip
+        {...defaultProps}
+        open={false}
+        PopperProps={{
+          popperRef,
+        }}
+      />,
+    );
+    setProps({ open: true });
+    const render1 = popperRef.current;
+    setProps({ open: true });
```

Comment:
Could you re-render this without setting props of the actual component? It's not clear if this only works because of `useMemo(factory)` or `memo(Component)`. Also: `firstPopperInstance` instead of `render1`?

Reply:
OK

Reply:
I have tried a new method to solve this problem (in case we face the same one in the future).